frontend/src/components/Transactions/TransactionCreation.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div :class="visible ? 'bg-secondary' : ''" class="transaction-slot-creation">
+    <div @click="visible = !visible">
+      <!-- Collaps Icon  -->
+      <div class="text-right" style="width: 95%; position: absolute">
+        <b-icon
+          :icon="visible ? 'caret-up-square' : 'caret-down-square'"
+          :class="visible ? 'text-black' : 'text-muted'"
+        />
+      </div>
+
+      <div>
+        <b-row>
+          <!-- ICON  -->
+          <b-col cols="1">
+            <div class="gdd-transaction-list-item-icon">
+              <b-icon icon="gift" class="gradido-global-color-accent m-mb-1 font2em" />
+            </div>
+          </b-col>
+
+          <b-col cols="11">
+            <!-- Betrag / Name Email -->
+            <b-row>
+              <b-col cols="5">
+                <div class="text-right">
+                  <span class="gdd-transaction-list-item-operator">+</span>
+                  <span class="gdd-transaction-list-item-amount">
+                    {{ $n(amount, 'decimal') }}
+                  </span>
+                </div>
+              </b-col>
+              <b-col cols="7">
+                <div class="gdd-transaction-list-item-name">
+                  {{
+                    typeId !== 'DECAY'
```

Comment:
this is always true?!

Reply:
diese datei ist gelöscht weil sie nicht verwendet wird